spec/controllers/admin/main_admin_controller_spec.rb

```diff
@@ -0,0 +1,35 @@
+describe Admin::MainAdminController do
+  describe '#index' do
+    subject { get_as user, :index }
+    shared_examples 'user with unauthorized access' do
+      it('should respond with 403') do
```

Comment:
Don't use parentheses with `it`. Also prefer using third-person present tense instead of "should", e.g. `it "responds with..."`